index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
```

Comment:
- [ ] As requested by the previous reviewer, the mobile version of the index page has no footer section. Kindly fix it.

Your design ends here
![image](https://user-images.githubusercontent.com/81932478/169658324-19ce86b7-ca24-4a5f-bef9-57752279bf03.png)

Provided design should include this.
![image](https://user-images.githubusercontent.com/81932478/169658364-64cf72f7-9d1f-4b86-ac8d-7c1ef36e4930.png)

### *I can understand the confusion because in the template for the index page it ends where you have ended but that is just to show that not all the speakers are shown upon page loading in mobile view.